legacy/src/api/bank-accounts/funds-transfers.md

```diff
@@ -165,6 +165,27 @@ A funds transfer represents either a top up to or a withdrawal from a Centrapay
 ]
 {% endjson %}
 
+## Abort Funds Transfer
```

Comment:
Should we put this at the bottom of the file? It's a bit weird that this is between the top up and withdrawal routes